plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/metastore/HiveMetastoreBackedDeltaLakeMetastore.java

```diff
@@ -118,6 +120,10 @@ public Optional<Table> getTable(String databaseName, String tableName)
     {
         Optional<Table> candidate = delegate.getTable(databaseName, tableName);
         candidate.ifPresent(table -> {
+            if (isPrestoView(table) && isHiveOrPrestoView(table)) {
```

Comment:
I just want to make sure that `&&` is correctly used in this context instead of `||`.

cc: @findepi @alexjo2144

Reply:
indeed, should probably be `if (isHiveOrPrestoView(...))` only